examples/v2/chat2.nim

```diff
@@ -49,6 +52,18 @@ type
   PrivateKey* = crypto.PrivateKey
   Topic* = waku_types.Topic
 
+
+# Similarly as Status public chats now.
```

Comment:
Does this mean that the symkey here is identical to one generated for "dingpu" chat in Waku v1 (generateFromPassword)? Maybe a good test, considering compatibility/bridging etc.

Reply:
Not exactly, hence the 'similarly" ;-).

I'd need to do this call not on the topic but on the full "channel name".